src/Reflection/ClassReflection.php

```diff
@@ -453,6 +467,15 @@ public function getMethod(string $methodName, ClassMemberAccessAnswerer $scope):
 		if ($scope->isInClass()) {
 			$key = sprintf('%s-%s', $key, $scope->getClassReflection()->getCacheKey());
 		}
+
+		if ($this->requireExtendsMethodsClassReflectionExtension->hasMethod($this, $methodName)) {
+			$method = $this->wrapExtendedMethod($this->requireExtendsMethodsClassReflectionExtension->getMethod($this, $methodName));
```

Comment:
We should not need to wrap it. The extension should already return the native method.

Reply:
Also you don't need to ask about "canCallMethod".

Reply:
it seems we cannot use the more specialized return type, because PHP 7.2/7.3 don't support covariance on the return type?

https://github.com/phpstan/phpstan-src/actions/runs/7431936254/job/20223170712?pr=2856

Reply:
Just write it in the PHPDoc.